### PR TITLE
Incorporation of LRS and update to Extreme Pathway Tests

### PR DESCRIPTION
**Pull request by tpfau**
This PR adds the lrs binaries in the COBRA.binary repository and updates the Extreme Pathway tests.
Reuse of the model from ExtremePathways to ExtremePools

**I hereby confirm that I have:**

- [X] Tested my code on my own machine
- [X] Followed the guidelines in the [Contributing Guide](https://opencobra.github.io/cobratoolbox/docs/contributing.html)
- [X] Selected `develop` as a target branch (top left drop-down menu)

*(Note: You may replace [] with [X] to check the box)*

### CODE REVIEW

File: src/base/install/setupCOBRASystemPath.m [hunk 1]
```diff
@@ -0,0 +1,31 @@
+function setupCOBRASystemPath()
```
Comment (laurentheirendt) [inline src/base/install/setupCOBRASystemPath.m]: Wouldn't it be better to rename this function to `setupCOBRAbinaryPath()` or similar?

File: src/analysis/topology/extremeRays/lrs/extremePathways.m [hunk 2]
```diff
@@ -99,20 +99,16 @@
...
-        % call lrs and wait until extreme pathways have been calculated
-        systemCallText = ['lrs ' pwd filesep filename '_' suffix '.ine > ' pwd filesep filename '_' suffix '.ext'];
-        [status, result] = system(systemCallText);
-        if status == 1
-            error(['lsr failed on file ', pwd filesep filename '_' suffix '.ine']);
-        end
-    else
-        error('lrs is not installed on your machine or not in the path. Refer to http://cgm.cs.mcgill.ca/~avis/C/lrs.html to install lrs.');
+[status, result] = system('which lrs');
+if ~isempty(strfind(result, '/lrs'))
```
Comment (laurentheirendt) [inline src/analysis/topology/extremeRays/lrs/extremePathways.m]: May you use here `filesep`?

File: src/base/install/setupCOBRASystemPath.m (hunk 1 above)
Comment (tpfau) [reply src/base/install/setupCOBRASystemPath.m]: probably best would be "addCOBRABinaryPathToSystemPath" (even though thats quite lengthy), but would actually say what it does.

File: src/analysis/topology/extremeRays/lrs/extremePathways.m (hunk 2 above)
Comment (tpfau) [reply src/analysis/topology/extremeRays/lrs/extremePathways.m]: Nope. 
`which` (also on windows) returns / as file separator. I was puzzled as well, but see the comments in the other files where this is looked up.

File: src/base/install/setupCOBRASystemPath.m (hunk 1 above)
Comment (tpfau) [reply src/base/install/setupCOBRASystemPath.m]: Would that be fine with you?

File: src/analysis/topology/extremeRays/lrs/extremePathways.m (hunk 2 above)
Comment (laurentheirendt) [reply src/analysis/topology/extremeRays/lrs/extremePathways.m]: Interesting ... may you put that as a note?

File: src/base/install/setupCOBRASystemPath.m (hunk 1 above)
Comment (laurentheirendt) [reply src/base/install/setupCOBRASystemPath.m]: Sounds good 👍

**Comment by laurentheirendt** [comment]
Thanks @tpfau 👍

**Pull request merged**